packages/amplify-util-mock/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "amplify-category-api-util-mock",
-  "version": "5.1.14",
+  "version": "6.0.0",
   "description": "Amplify CLI plugin providing local testing",
   "repository": {
     "type": "git",
```

Comment:
This one `amplify-category-api-util-mock` will still carry CDK v1 references through `amplify-provider-awscloudformation` from CLI repo...
Which means we need two (?) cycles of data - cli tag releases (and then normal releases at some point) to get this straight.

See 

```
$ yarn why @aws-cdk/core
yarn why v1.18.0
[1/4] 🤔  Why do we have the module "@aws-cdk/core"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "@aws-cdk/core@1.124.0"
info Reasons this module exists
   - "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-auth-transformer" depends on it
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-auth-transformer#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-function-transformer#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-http-transformer#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-index-transformer#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-model-transformer#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-relational-transformer#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-transformer-core#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-transformer-interfaces#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-util-mock#amplify-provider-awscloudformation#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-util-mock#amplify-provider-awscloudformation#@aws-cdk#assert#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-util-mock#amplify-provider-awscloudformation#@aws-cdk#aws-apigatewayv2#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-model-transformer#@aws-cdk#aws-applicationautoscaling#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-function-transformer#@aws-cdk#aws-appsync#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-util-mock#amplify-provider-awscloudformation#@aws-cdk#aws-autoscaling#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-transformer-core#@aws-cdk#aws-certificatemanager#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-model-transformer#@aws-cdk#aws-cloudwatch#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-transformer-core#@aws-cdk#aws-codeguruprofiler#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-transformer-core#@aws-cdk#aws-cognito#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-index-transformer#@aws-cdk#aws-dynamodb#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-model-transformer#@aws-cdk#aws-ec2#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-util-mock#amplify-provider-awscloudformation#@aws-cdk#aws-ecr-assets#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-util-mock#amplify-provider-awscloudformation#@aws-cdk#aws-ecr#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-util-mock#amplify-provider-awscloudformation#@aws-cdk#aws-ecs#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-transformer-core#@aws-cdk#aws-efs#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-util-mock#amplify-provider-awscloudformation#@aws-cdk#aws-elasticloadbalancing#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-util-mock#amplify-provider-awscloudformation#@aws-cdk#aws-elasticloadbalancingv2#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-transformer-core#@aws-cdk#aws-elasticsearch#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-transformer-core#@aws-cdk#aws-events#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-model-transformer#@aws-cdk#aws-iam#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-model-transformer#@aws-cdk#aws-kms#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-function-transformer#@aws-cdk#aws-lambda#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-transformer-core#@aws-cdk#aws-logs#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-transformer-interfaces#@aws-cdk#aws-rds#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-transformer-core#@aws-cdk#aws-route53#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-model-transformer#@aws-cdk#aws-s3-assets#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-transformer-core#@aws-cdk#aws-s3#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-transformer-interfaces#@aws-cdk#aws-secretsmanager#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-util-mock#amplify-provider-awscloudformation#@aws-cdk#aws-servicediscovery#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-util-mock#amplify-provider-awscloudformation#@aws-cdk#aws-sns#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-transformer-core#@aws-cdk#aws-sqs#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-model-transformer#@aws-cdk#custom-resources#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-util-mock#amplify-provider-awscloudformation#@aws-amplify#cli-extensibility-helper#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-util-mock#amplify-provider-awscloudformation#@aws-amplify#graphql-transformer-core#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-util-mock#amplify-provider-awscloudformation#@aws-amplify#graphql-transformer-interfaces#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-util-mock#amplify-provider-awscloudformation#@aws-cdk#aws-ecr-assets#@aws-cdk#assets#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-util-mock#amplify-provider-awscloudformation#@aws-amplify#cli-extensibility-helper#@aws-cdk#aws-apigateway#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-model-transformer#@aws-cdk#aws-applicationautoscaling#@aws-cdk#aws-autoscaling-common#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-util-mock#amplify-provider-awscloudformation#@aws-cdk#aws-ecs#@aws-cdk#aws-autoscaling-hooktargets#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-model-transformer#@aws-cdk#custom-resources#@aws-cdk#aws-cloudformation#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-util-mock#amplify-provider-awscloudformation#@aws-cdk#aws-sns#@aws-cdk#aws-codestarnotifications#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-index-transformer#@aws-cdk#aws-dynamodb#@aws-cdk#aws-kinesis#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-util-mock#amplify-provider-awscloudformation#@aws-cdk#aws-ecs#@aws-cdk#aws-route53-targets#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-transformer-interfaces#@aws-cdk#aws-secretsmanager#@aws-cdk#aws-sam#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-function-transformer#@aws-cdk#aws-lambda#@aws-cdk#aws-signer#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-graphql-migration-tests#@aws-amplify#graphql-model-transformer#@aws-cdk#aws-ec2#@aws-cdk#aws-ssm#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-util-mock#amplify-category-function#amplify-cli-core#@aws-amplify#graphql-transformer-core#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-util-mock#amplify-category-function#amplify-cli-core#@aws-amplify#graphql-transformer-interfaces#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-util-mock#amplify-provider-awscloudformation#@aws-cdk#aws-ecs#@aws-cdk#aws-route53-targets#@aws-cdk#aws-cloudfront#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-util-mock#amplify-provider-awscloudformation#@aws-cdk#aws-ecs#@aws-cdk#aws-route53-targets#@aws-cdk#aws-globalaccelerator#@aws-cdk#core"
   - Hoisted from "_project_#amplify-category-api-util-mock#amplify-provider-awscloudformation#@aws-cdk#aws-ecs#@aws-cdk#aws-autoscaling-hooktargets#@aws-cdk#aws-sns-subscriptions#@aws-cdk#core"
info Disk size without dependencies: "4.56MB"
info Disk size with unique dependencies: "7.49MB"
info Disk size with transitive dependencies: "8.09MB"
info Number of shared dependencies: 17
✨  Done in 0.61s.
```

Reply:
IIUC , We can convert cli to use cdkV2 , but this will fetch cdkV1 dependencies from api Repo.

Then a data release removing cdkV1 dependencies , this will flush out cdkV1 dependencies right.